Source/ACE.Server/Network/GameEvent/Events/GameEventPlayerDescription.cs

```diff
@@ -299,6 +299,9 @@ private void WriteEventBody()
             if (shortcuts.Count > 0)
                 optionFlags |= CharacterOptionDataFlag.Shortcut;
 
+            if (Session.Character.GameplayOptions.Length > 0)
```

Comment:
Null check needed on Session.Character.GameplayOptions.Length.  Other than this one item, PR works great.